charts/node-server/templates/mongodb-deploy.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
         - name: mongodb
-          image: mongo:5.0.11
+          image: docker.io/library/mongo:5.0.11
```

Comment:
Did you do it because of the endpoint definition?
Was it required, or just in case?

Reply:
when you don't specify a path to the image `docker.io/library` is assumed
also `something/image` is assumed to be `docker.io/something/image`
You can see this your self by looking at an output of a pull:
```
$ docker pull mongo
Using default tag: latest
latest: Pulling from library/mongo
7a9f619ee5e9: Already exists
fe07d8ebcafb: Already exists
01f116473d70: Already exists
5ed58e863f95: Already exists
62274e6e1246: Already exists
fe64d6112f44: Pull complete
baed58b85549: Pull complete
18825bc51a8f: Pull complete
04c35fa85a1d: Pull complete
Digest: sha256:2b527cb8eafb4a97a896eb05a2e88c75f0033d4be4bf94fad205299659147c13
Status: Downloaded newer image for mongo:latest
docker.io/library/mongo:latest
```

Reply:
The reason I've added this (and also updated 2 more images that weren't in the full spec) is because I'm later creating the `images.txt` files with all of them and then running regular expressions in the script:
```bash
$ image=docker.io/library/rabbitmq:3.9.22
$ registry_port=5699
$ target=$(echo $image | sed "s/[^\/]*\//127.0.0.1:$registry_port\//" | sed 's/@.*$//')
$ api_url=$(echo $target | sed 's/\//\/v2\//' | sed 's/:/\/manifests\//2')
$ echo $target
127.0.0.1:5699/library/rabbitmq:3.9.22
$ echo $api_url
127.0.0.1:5699/v2/library/rabbitmq/manifests/3.9.22

```

It's going to be a lot harder improving the scripts to support partial image names instead of just putting to full path there (which is also considered best practice in all the other helm charts we use)